opencti-platform/opencti-front/src/private/components/common/form/ObjectAssigneeField.tsx

```diff
@@ -102,17 +103,24 @@ const ObjectAssigneeField: FunctionComponent<ObjectAssigneeFieldProps> = ({
       .then((data) => {
         const newAssignees = (
           (data as ObjectAssigneeFieldMembersSearchQuery$data)?.members?.edges ?? []
-        ).map((n) => ({
-          label: n.node.name,
-          value: n.node.id,
-          type: n.node.entity_type,
-        })).sort((a, b) => {
+        ).map((n) => {
+          const group = n.node.id === me?.id ? t_i18n('User') : t_i18n('All');
+          return {
+            label: n.node.name,
+            value: n.node.id,
+            type: n.node.entity_type,
+            group,
+          };
+        }).sort((a, b) => {
           // Display first the current user
           if (a.value === me?.id) return -1;
           if (b.value === me?.id) return 1;
           // Sort by alphabetic order
           return a.label.localeCompare(b.label);
         });
+        // Add current user if is not in the only first results displayed
```

Comment:
Shouldn't we add the current user before the 'display first the current user' part ?

Reply:
updated